library/src/scripts/embeddedContent/PanoptoEmbed.tsx

```diff
@@ -17,8 +17,12 @@ interface IProps extends IBaseEmbedProps {
     height: number;
 }
 
+/** @constant {string} */
 const PANOPTO_SCRIPT = "https://developers.panopto.com/scripts/embedapi.min.js";
 
+/** @constant {string} */
+const EMBED_LOADED_CLASS = "isLoaded";
+
```

Comment:
This is a bit odd. We tend not to use JSDoc for our types. Better to enforce with typescript if you want to manually impose some types.

```suggestion
const PANOPTO_SCRIPT: string = "https://developers.panopto.com/scripts/embedapi.min.js";

const EMBED_LOADED_CLASS: string = "isLoaded";

```

Reply:
Oh ok I didn't know that, I'll make the change.